MapleServer2/Tools/GameCommandActions.cs

```diff
@@ -170,5 +176,28 @@ private static bool TryParseCoord(string s, out CoordF result)
             result = default;
             return false;
         }
+
+        private static long ParseLong(GameSession session, string s)
+        {
+            long temp = -1;
```

Comment:
Now you don't need this temp since you returned on line 185, you can just return `-1` directly